src/ocrd_network/constants.py

```diff
@@ -1,2 +1,55 @@
-NETWORK_AGENT_SERVER = 'server'
-NETWORK_AGENT_WORKER = 'worker'
+from enum import Enum
+
+DOCKER_IMAGE_MONGO_DB = "mongo"
+DOCKER_IMAGE_RABBIT_MQ = "rabbitmq:3.12-management"
+# These feature flags are required by default to use the newer version
+DOCKER_RABBIT_MQ_FEATURES = "quorum_queue,implicit_default_bindings,classic_mirrored_queue_version"
+
+NETWORK_API_TAG_ADMIN = "admin"
```

Comment:
make these enums as well, while you're at it?

Reply:
Network API tags, sure. Still not sure about docker images and features though.